### PR TITLE
Fix/recover_tests_report

### PR DESCRIPTION
**Pull request by ziiiki**
Tests wont pass because the code is not adapted to Slurm-API integrated backend. 

See https://github.com/qilimanjaro-tech/qiboconnection/pull/135, which points to this PR, for qiboconnection and end2end tests adapted to Slurm. Hence, once is merged, this tests will pass

### CODE REVIEW

**Comment by AlbertMitjans** [comment]
Hello. You may have forgotten to update the changelog!
Please edit [`changelog-dev.md`](/qilimanjaro-tech/qiboconnection/tree/main/changelog-dev.md) with:
* A one-to-two sentence description of the change. You may include a small working example for new features.
* A link back to this PR.

File: tests/end2end/logs/report.html
Comment (JavierSab) [inline tests/end2end/logs/report.html:1]: probably this file is not supposed to be commited

Comment (JavierSab) [reply tests/end2end/logs/report.html:1]: (and propably neither is the rest of files under /logs)

File: tests/end2end/tests/test_circuits.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 
 import pytest
 from qibo.models import Circuit
 
 from qiboconnection.api import API
 from qiboconnection.models.devices import Device
+from tests.end2end.utils.operations import Operation, check_operation_possible_or_skip
+from tests.end2end.utils.utils import delete_job, get_devices_listing_params, post_and_get_result
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "..")))
```
Comment (JavierSab) [inline tests/end2end/tests/test_circuits.py:15]: oh, I see... We can have a look at this later, but it is what it is ¯\_(ツ)_/¯

**Review by JavierSab — APPROVED** [review]
This time comments are really comments, not changes. Lets go with this and see how the other part behaves!

File: tests/end2end/logs/report.html
Comment (ziiiki) [reply tests/end2end/logs/report.html:1]: true, that was an error

**Pull request merged**